benchmarks/buiter_2016_wedge/buiter_2016_wedge.prm

```diff
@@ -0,0 +1,172 @@
+# This is the input file for the setup for the benchmark described in Buiter et al, 2016
+# Experiment 1 which simulates a stable wedge undergoing translation.
+
+
+#####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver tolerance             = 1e-9
+set Max nonlinear iterations               = 25
+set CFL number                             = 0.5
+set Maximum time step                      = 1e6
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+#### Parameters describing the model
+
+# Model geometry
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 500
+    set Y repetitions = 100
+    set X extent      = 0.15
+    set Y extent      = 0.035
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left
+  set Prescribed velocity boundary indicators = right:function
+  set Zero velocity boundary indicators = bottom
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = cm=0.01, h=3600., th=0.002
```

Comment:
Should `h` be `3500.` instead of `3600.`? Also what does `th` stand for? You can indicate this by either expanding the variable name or adding a few comments at the top of the section.